sim/rogue/crimson_tempest.go

```diff
@@ -92,5 +92,8 @@ func (rogue *Rogue) registerCrimsonTempestSpell() {
 }
 
 func (rogue *Rogue) CrimsonTempestDamage(comboPoints int32) float64 {
-	return []float64{0.15, 0.3, 0.45, 0.6, 0.75, 0.9}[comboPoints] * rogue.GetStat(stats.AttackPower)
+    tickDamageValues := []float64{0, 0.3, 0.45, 0.6, 0.75, 0.9}
+    tickDamage := tickDamageValues[comboPoints] * rogue.GetStat(stats.AttackPower)/float64(comboPoints+2)
```

Comment:
> /float64(comboPoints+2)

Should this be comboPoints +1? A 5 combo point Crimson Tempest ticks 6 times right?

Reply:
You could probably also pass the number of ticks in from the DoT struct when calling this